src/RecoveryServices/RecoveryServices/help/Get-AzRecoveryServicesBackupContainer.md

```diff
@@ -78,7 +78,7 @@ The acceptable values for this parameter are:
 Type: Microsoft.Azure.Commands.RecoveryServices.Backup.Cmdlets.Models.ContainerType
 Parameter Sets: (All)
 Aliases:
-Accepted values: AzureVM, Windows, AzureSQL, AzureStorage, AzureWorkload
+Accepted values: AzureVM, Windows, AzureSQL, AzureStorage, AzureVMAppContainer
```

Comment:
This looks like a breaing change.  Is the `AzureWorkload` value no longer supported?